proteuslib/edb/tests/data.py

```diff
@@ -13,135 +13,636 @@
+        "pressure": (5e4, 1e5, 1e6, pyunits.Pa),
+    },
+    "pressure_ref": (101325, pyunits.Pa),
+    "temperature_ref": (300, pyunits.K),
+    # Defining phase equilibria
+    "phases_in_equilibrium": [("Vap", "Liq")],
+    "phase_equilibrium_state": {("Vap", "Liq"): SmoothVLE},
+    "bubble_dew_method": IdealBubbleDew,
+}
+
+recarbonation_reaction_config = {
+    "base_units": {
+        "time": pyunits.s,
+        "length": pyunits.m,
+        "mass": pyunits.kg,
+        "amount": pyunits.mol,
+        "temperature": pyunits.K,
+    },
+    "equilibrium_reactions": {
+        "H2O_Kw": {
+            "stoichiometry": {
+                ("Liq", "H2O"): -1,
+                ("Liq", "H_+"): 1,
+                ("Liq", "OH_-"): 1,
             },
-            "enth_mol_form_liq_comp_ref": [-542.83, "U.J/U.mol"],
-            "cp_mol_liq_comp_coeff": {
-                "1": [276370.0, "U.J/U.kmol/U.K"],
-                "2": [-2090.1, "U.J/U.kmol/U.K**2"],
-                "3": [8.125, "U.J/U.kmol/U.K**3"],
-                "4": [-0.014116, "U.J/U.kmol/U.K**4"],
-                "5": [9.3701e-06, "U.J/U.kmol/U.K**5"],
+            "heat_of_reaction": constant_dh_rxn,
+            "equilibrium_constant": gibbs_energy,
+            "equilibrium_form": log_power_law_equil,
+            "concentration_form": ConcentrationForm.molarity,
+            "parameter_data": {
+                "dh_rxn_ref": (55.830, pyunits.kJ / pyunits.mol),
+                "ds_rxn_ref": (-80.7, pyunits.J / pyunits.mol / pyunits.K),
+                "T_eq_ref": (300, pyunits.K),
+                # By default, reaction orders follow stoichiometry
+                #    manually set reaction order here to override
+                "reaction_order": {
+                    ("Liq", "H2O"): 0,
+                    ("Liq", "H_+"): 1,
+                    ("Liq", "OH_-"): 1,
+                },
+            }
+            # End parameter_data
+        },
+        # End R1
+        "CO2_to_H2CO3": {
+            "stoichiometry": {
+                ("Liq", "H2O"): -1,
+                ("Liq", "CO2"): -1,
+                ("Liq", "H2CO3"): 1,
             },
-            "entr_mol_form_liq_comp_ref": [-53, "U.J/U.K/U.mol"],
+            "heat_of_reaction": constant_dh_rxn,
+            "equilibrium_constant": van_t_hoff,
+            "equilibrium_form": log_power_law_equil,
+            "concentration_form": ConcentrationForm.molarity,
+            "parameter_data": {
+                "dh_rxn_ref": (0, pyunits.kJ / pyunits.mol),
+                "k_eq_ref": (1.7 * 10 ** -3, None),
+                "T_eq_ref": (300, pyunits.K),
+                # By default, reaction orders follow stoichiometry
+                #    manually set reaction order here to override
+                "reaction_order": {
+                    ("Liq", "H2CO3"): 1,
+                    ("Liq", "CO2"): -1,
+                    ("Liq", "H2O"): 0,
+                },
+            }
```

Comment:
This is correct configuration for the 'CO2_to_H2CO3' reaction on a <code>Concentration.molarity</code> basis. 

All of your configs from line 203 (recarbonation_thermo_config) and downward should be correct for both thermo and reaction configurations.